com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -580,11 +745,20 @@ half3 VertexLighting(float3 positionWS, half3 normalWS)
 //       Used by ShaderGraph and others builtin renderers                    //
 ///////////////////////////////////////////////////////////////////////////////
 half4 UniversalFragmentPBR(InputData inputData, half3 albedo, half metallic, half3 specular,
-    half smoothness, half occlusion, half3 emission, half alpha)
+    half smoothness, half occlusion, half3 emission, half alpha
+#ifdef _CLEARCOAT
```

Comment:
For future, we will most likely want to add extra parameters to this function. What if we add an override now that take the SurfaceInput function that already has the new clear coat params. Then we can add parameters to the struct as needed without having to change the signature for the function in the future?

Reply:
I did not do this yet. It was unclear why this was expanded in the first place. I think in LWRP it was a struct. I left it for later after I more familiar with shadergraph which might have some dependencies.

Reply:
done.